src/Login.js

```diff
@@ -15,7 +15,7 @@ class Login {
   }
   
   validateCustomerId() {
-    let number = Number(this.username.trim().slice(-2));
+    let number = Number(this.username.trim().slice(8));
     return number > 0 && number <= 50;
   }
 
```

Comment:
Awesome, such a small detail to make it dynamic. I know I know, negative indexes are pretty cool but it just wasn't doing what we needed.